.github/workflows/build.yml

```diff
@@ -187,3 +189,30 @@ jobs:
         run: make init
       - name: Run functional & smoke tests
         run: pytest -vv -n 4 tests/functional tests/smoke
+
+  docker-disabled:
+    name: docker-disabled-test
+    if: github.repository_owner == 'aws'
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-python@v4
+        with:
+          python-version: |
```

Comment:
Can we comment this is runtime supported versions not SAM CLI installation versions?

Also worth making sure our "adding/removing runtime support" docs are updated so we remember this needs to be updated.